sdk/python/tests/api/test_integration.py

```diff
@@ -240,7 +240,7 @@ async def test_directory_sync():
 async def test_deprecation_warning():
     async with dagger.Connection() as client:
         with pytest.warns(DeprecationWarning, match="with_exec"):
-            await client.container().from_("alpine:3.16.2").exec(["true"])
+            await client.container().from_("alpine:3.16.2").exec(args=["true"])
```

Comment:
Following this PR: https://github.com/dagger/dagger/commit/22d1ca63bcf7e0b3d05b9c491b74e12c8de45688. Shouldn't we try to bump all alpine versions to [alpine:3.18.2](https://github.com/dagger/dagger/commit/22d1ca63bcf7e0b3d05b9c491b74e12c8de45688#diff-5e1df8cb218aca17cfd229832a2f41f46d4c1825d55c006500945c737c047aa9R16)

Reply:
Yeah, best do it in another PR. This particular test is going away though. Not sure how I'm going to test a deprecation without deprecations after https://github.com/dagger/dagger/issues/5479. 🙂